src/setup/cortx_deploy

```diff
@@ -110,6 +111,46 @@ node_id=`cat /etc/machine-id`
 cluster_id=$(conf_get $(yaml_config $CLUSTER_INFO) "cluster>id")
 log $INFO "Starting provisioning on node $node_id which is part of cluster $cluster_id."
 
+deploy=false
```

Comment:
Lets not introduce new flags... Ideally we should leverage on using the same flags as mentioned in:
src/provisioner/provisioner.py

`class PROVISIONING_STAGES(Enum):`
`class PROVISIONING_STATUS(Enum):`

Reply:
Hi Nitin,
This is a shell script outside the provisioner.py python file.